crypto/core_namemap.c

```diff
@@ -373,45 +373,47 @@ int ossl_namemap_add_names(OSSL_NAMEMAP *namemap, int number,
 #include <openssl/evp.h>
 
 /* Creates an initial namemap with names found in the legacy method db */
-static void get_legacy_evp_names(const char *main_name, const char *alias,
+static void get_legacy_evp_names(const char *name, const char *desc,
                                  void *arg)
 {
-    int main_id = ossl_namemap_add_name(arg, 0, main_name);
+    int num = ossl_namemap_add_name(arg, 0, name);
 
     /*
-     * We could check that the returned value is the same as main_id,
-     * but since this is a void function, there's no sane way to report
-     * the error.  The best we can do is trust ourselve to keep the legacy
-     * method database conflict free.
+     * We currently treat the description ("long name" in OBJ speak) as an
+     * alias.
+     * TODO: Change the treatment of descriptions to be separate, so it can
```

Comment:
create an issue if this is a problem.. Remove TODO

Reply:
I actually did, #14514.  I'm constantly worried that I'll forget where I found the issue in the code, but I won't forget this one, so ok.